exo-add/features/exoservice-es6-mongodb.feature

```diff
@@ -54,7 +54,7 @@ Feature: scaffolding an ExoService written in ES6, backed by MongoDB
           - user.updated
 
       docker:
-        link:
+        dependencies:
```

Comment:
The service just declares that it has a dependency on "MongoDB". In dev mode that's provided as a Docker image. In production it is provided as a link to an external, possibly third-party system. So this really isn't a Docker thing at all, it's just coincidentally Docker in dev mode.